src/super_context.ml

```diff
@@ -33,6 +33,245 @@ module Env_node = struct
     }
 end
 
+module Var : sig
```

Comment:
Yeah, I agree, but this module is really private to super contexts. Can you think of a good name for this? Longer term, I'm thinking of renaming `String_with_vars` to `Template` and then it will be possible to have a `Var` module.

By the way, this really highlights the need to have sub library like modules that we've talked about.

Reply:
So for now I'll create make `var.ml` for this module.